reference/orders.v2.oas2.yml

```diff
@@ -4720,5 +4954,343 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/shippingAddress_Base'
+            consignments:
+              $ref: '#/components/schemas/orderConsignment_Post'
         - $ref: '#/components/schemas/order_Shared'
       x-internal: false
+    orderConsignment_Get:
+      title: orderConsignment_Get
+      type: object
+      properties:
+        shipping:
+          type: array
+          items:
+            $ref: '#/components/schemas/shippingConsignment_Get'
+        downloads:
+          type: array
+          items:
+            $ref: '#/components/schemas/digitalConsignment_Get'
+          maxItems: 1
+        email:
+          type: object
+          properties:
+            gift_certificates:
+              type: array
+              items:
+                $ref: '#/components/schemas/giftCertificateConsignment_Get'
+    shippingConsignment_Post:
+      allOf:
+        - $ref: '#/components/schemas/shippingConsignment_Base'
+        - type: object
+          properties:
+            line_items:
+              type: array
+              minItems: 1
+              items:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/orderCustomProduct_Post'
+                    - $ref: '#/components/schemas/orderCatalogProduct_Post'
+          required:
+            - line_items
```

Comment:
Are `line_items` really the only properties in a shipping consignment that are required? From my own testing at least the `country`/`country_iso2 ` and `zip` need to be provided or we will fail to resolve the tax zone. Potentially we may want to force them to populate other fields for a "full" address too.
In the CP we force all the following to be provided:
- full name
- last name
- street_1
- city
- country
- zip

Reply:
Thanks @BC-SEven yes, `country/country_iso2` and `zip` need to be populated. 
It's a good question as to whether we want to add additional checks
The current ones are based on what we do for the legacy payload, but it may well be worth tightening them up for the new consignment payload

Reply:
https://bigcommercecloud.atlassian.net/browse/ORDERS-5089